src/main/index.ts

```diff
@@ -74,6 +74,10 @@ function init() {
 
         registerScreenShareHandler();
         registerMediaPermissionsHandler();
+        //register file handler so we can load the custom splash animation from the user's filesystem
+        protocol.handle("splash-animation", () => {
+            return net.fetch("file:///"+splashAnimationPath);
```

Comment:
Use `fs.readSync` instead

Reply:
Because it doesn't make sense to have a "network request" to grab a local file when you can just read it directly